tasks/configure.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+- name: Set Amazon Inspector Agent Service State
+  become: yes
+  service:
+    name: awsagent
+    state: "{{ sansible_amazon_inspector_agent_state }}"
+    enabled: "{{ sansible_amazon_inspector_agent_enabled }}"
+  when:
+    - not sansible_amazon_inspector_agent_download_only
+    - ansible_user_id != "vagrant"
```

Comment:
We try to keep Vagrant stuff out of these files, can you rely on the role flags only here?